### PR TITLE
[WIP] Use OIC Data Models

### PR DESCRIPTION
**Pull request by bdilly**
It still fails on some points when building.

I wasn't able to check all issues yet.

### CODE REVIEW

**Comment by otaviobp** [comment]
looks good to me so far

File: src/modules/flow/oic/Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 obj-$(FLOW_NODE_TYPE_OIC) += oic.mod
 obj-oic-$(FLOW_NODE_TYPE_OIC) := oic.json oic.o
-obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := core.light.json
+obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := y
+obj-oic-$(FLOW_NODE_TYPE_OIC)-oic-dirs := $(OIC_DATA_MODELS)
```
Comment (glima) [inline src/modules/flow/oic/Makefile]: We can live with just one of both (new) variables, no?

Comment (glima) [inline src/modules/flow/oic/Makefile]: OK, that would involve changes in the makefile rule. I can tweak that later.

Comment (bdilly) [inline src/modules/flow/oic/Makefile]: If there is any way to represent the current dir all of them could be used on oic-dirs.

But I believe just using

obj-oic-$(FLOW_NODE_TYPE_OIC)-oic := y

would be more frequent than including extra dirs anyway.

**Comment by bdilly** [comment]
New version pushed: #1781

**Pull request closed without merging**